src/common/http_header.cc

```diff
@@ -204,7 +203,6 @@ CacheControl::write(std::ostream& os) const {
             case CacheDirective::SMaxAge:
                 return "s-maxage";
             case CacheDirective::Ext:
-                return "";
```

Comment:
Don't rely on fall-through here.
I am also surprise that there is a default, since directive.directive() is an enum (or I thought that it was?).

Reply:
I am surprised that there is a lambda function instead of a map. =P

Reply:
But this is not the point. The effort wasn't in refactoring the code just improving coverage of the test case.

Reply:
Reverted unnecessary modifications to http_header.cc (just to coverage point of view).